app/backend/gwells/migrations/0009_auto_20240605_1836.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2 on 2024-06-05 18:36
```

Comment:
WARNINGS:
backend-1  | gwells.Border: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
backend-1  | 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the GWellsConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
backend-1  | DEBUG 2024-06-12 18:07:14,660 selector_events.py[58]:selector_events::__init__ Using selector: EpollSelector
backend-1  | Error in update_well for Well ID 100123: Well matching query does not exist.
backend-1  | Error in update_well for Well ID 100124: Well matching query does not exist.
backend-1  | Error in update_well for Well ID 100125: Well matching query does not exist.
backend-1  | Error in update_well for Well ID 117042: Well matching query does not exist.